lib/bolt/applicator.rb

```diff
@@ -90,7 +90,7 @@ def compile(target, ast, plan_vars)
         pdb_config: @pdb_client.config.to_hash,
         hiera_config: @hiera_config,
         target: {
-          name: target.host,
+          name: target.safe_name,
```

Comment:
Since this and the trusted info is actually used in code I think it needs to be the real name.